vllm/model_executor/layers/sparsity/semi_structured_sparse_w16a16.py

```diff
@@ -0,0 +1,49 @@
+import torch
+
+from typing import Any, Dict, List, Type
+from magic_wand import CompressedStorageFormat
+from vllm.model_executor.layers.sparsity.base_config import SparsityConfig
+from .sparse_w16a16_linear_method import SparseW16A16LinearMethod
+from magic_wand import (
+    CompressedStorageFormat,
+    SparseSemiStructuredStorageFormat
+)
+
+class SemiStructuredSparseW16A16Config(SparsityConfig):
+    """Config class for SemiStructuredSparseW16A16.
+    """
+
+    def __init__(self) -> None:
+        # TODO: Add new configs here
+        pass
+
+    def __repr__(self) -> str:
+        return "SemiStructuredSparseW16A16Config()"
+
+    @classmethod
+    def get_storage_format_cls(cls) -> Type[CompressedStorageFormat]:
+        return SparseSemiStructuredStorageFormat
+
+    @classmethod
+    def get_name(cls) -> str:
+        return "semi_structured_sparse_w16a16"
+
+    @classmethod
+    def get_supported_act_dtypes(cls) -> List[torch.dtype]:
+        return [torch.half]
```

Comment:
is there not support for bf16?

Reply:
I pinged @tlrmchlsmth about whether to also prioritize bfloat16, it is not necessarily a huge ask, just something I need to jump in and test. I only prioritized & tested torch.half, not bfloat16.

Reply:
Update: bfloat16 support is desirable, I think it makes sense to be in a separate PR.